samples/subsys/fs/littlefs/boards/stm32h747i_disco_m7_blk.conf

```diff
@@ -0,0 +1,3 @@
+CONFIG_SDMMC_STM32_HWFC=y
+CONFIG_FS_LITTLEFS_FC_HEAP_SIZE=2048
+CONFIG_MAIN_STACK_SIZE=2048
```

Comment:
Since there is no board dependency, this file could be renamed as stm32_blk.cnf and use for several boards.
Once this is done this can allow some further factorization in the test configurations.